antarest/study/business/aggregator_management.py

```diff
@@ -54,28 +78,80 @@ def _checks_estimated_size(nb_files: int, df_bytes_size: int, nb_files_checked:
         raise FileTooLargeError(estimated_df_size, maximum_size)
 
 
+def _columns_ordering(df_cols: t.List[str], column_name: str, is_details: bool, mc_root: MCRoot) -> t.Sequence[str]:
+    org_cols = df_cols.copy()
+    if mc_root == MCRoot.MC_ALL:
+        org_cols = [
+            col for col in org_cols if col not in {column_name, CLUSTER_ID_COL, MCYEAR_COL, TIME_ID_COL, TIME_COL}
+        ]
+    if is_details:
+        org_cols = [col for col in org_cols if col != CLUSTER_ID_COL and col != TIME_ID_COL]
+    if mc_root == MCRoot.MC_IND:
+        new_column_order = (
+            [column_name] + ([CLUSTER_ID_COL] if is_details else []) + [MCYEAR_COL, TIME_ID_COL, TIME_COL] + org_cols
+        )
+    elif mc_root == MCRoot.MC_ALL:
+        new_column_order = [column_name] + ([CLUSTER_ID_COL] if is_details else []) + [TIME_ID_COL, TIME_COL] + org_cols
```

Comment:
I think it would be easier to read if you'd construct the list with MCYEAR_COL inside and then if mc_root == MCRoot.MC_IND: new_column_order.remove(MCYEAR_COL)

Reply:
I will do something different